src/MediaManagerBundle/Bridge/Chameleon/BackendModule/MediaManagerBackendModule.php

```diff
@@ -386,7 +386,7 @@ public function GetHtmlFooterIncludes()
     {
         $includes = parent::GetHtmlFooterIncludes();
         $includes[] = '<script src="'.TGlobal::GetStaticURL(
-                '/bundles/chameleonsystemmediamanager/lib/jstree/3.3.2/jstree.js'
+                '/bundles/chameleonsystemmediamanager/lib/jstree/3.3.7/jstree.js'
```

Comment:
In line 362 the version needs to be adjusted too, right?

Reply:
Did this already.